docker/composable-sandbox.dockerfile

```diff
@@ -1,9 +1,22 @@
 FROM composablefi/ci-linux:2022-04-18 as builder
 
+# ISSUE: we already copied context when started build, what the heck we are roing here? it slows down builds
 COPY . /build
 WORKDIR /build
 
-RUN cargo build --release
+# NOTE: dirty hack until nix migration
+RUN cargo +nightly build --release -p wasm-optimizer
+RUN cargo +nightly build --release -p composable-runtime-wasm --target wasm32-unknown-unknown --features=runtime-benchmarks
+RUN cargo +nightly build --release -p picasso-runtime-wasm --target wasm32-unknown-unknown --features=runtime-benchmarks
+RUN cargo +nightly build --release -p dali-runtime-wasm --target wasm32-unknown-unknown --features=runtime-benchmarks
+RUN ./target/release/wasm-optimizer --input ./target/wasm32-unknown-unknown/release/dali_runtime.wasm --output ./target/wasm32-unknown-unknown/release/dali_runtime.optimized.wasm
+RUN ./target/release/wasm-optimizer --input ./target/wasm32-unknown-unknown/release/picasso_runtime.wasm --output ./target/wasm32-unknown-unknown/release/picasso_runtime.optimized.wasm
+RUN ./target/release/wasm-optimizer --input ./target/wasm32-unknown-unknown/release/composable_runtime.wasm --output ./target/wasm32-unknown-unknown/release/composable_runtime.optimized.wasm
+
+RUN export DALI_RUNTIME=$(realpath ./target/wasm32-unknown-unknown/release/dali_runtime.optimized.wasm) && \
+	export PICASSO_RUNTIME=$(realpath ./target/wasm32-unknown-unknown/release/picasso_runtime.optimized.wasm) && \
+	export COMPOSABLE_RUNTIME=$(realpath ./target/wasm32-unknown-unknown/release/composable_runtime.optimized.wasm) && \
+	cargo build --release --features=builtin-wasm
```

Comment:
Can we already include the nix migration in this PR?

Reply:
in theory can, so i would prefer strep by step migration. one TODO at a time from now. we replaced core and can go from there.

Reply:
actually it requires to fix all the way down then, including runs of all devs and runtime tests, etc. we can do as separate pr next.

Reply:
there are more important topic to solve, e.g. how to output side effects from flake (e.g. formatted files or benchmarks)